lib/active_shipping/carriers/usps.rb

```diff
@@ -432,20 +436,24 @@ def rates_from_response_node(response_node, packages, options = {})
         %w(Service ID SvcDescription)   << INTERNATIONAL_RATE_FIELD[commercial_type]
       end
 
-      root_node.each_element('Package') do |package_node|
-        this_package = packages[package_node.attributes['ID'].to_i]
+      root_node.xpath('Package').each do |package_node|
+        this_package = packages[package_node['ID'].to_i]
+
+        package_node.xpath(service_node).each do |service_response_node|
+          service_name = service_response_node.at(service_name_node).text
 
-        package_node.each_element(service_node) do |service_response_node|
-          service_name = service_response_node.get_text(service_name_node).to_s
+          # service_name.gsub!('&lt;sup&gt;&amp;reg;&lt;/sup&gt;', '') # '®'
+          # service_name.gsub!('&lt;sup&gt;&#8482;&lt;/sup&gt;', '') # '™'
+          # service_name.gsub!('&lt;sup&gt;&amp;trade;&lt;/sup&gt;', '') # '™'
+          # service_name.gsub!(/\*+\z/, '') # '**'
```

Comment:
This was me investigating some slight differences between Nokogiri and REXML. Forgot to remove it.